### ISSUE TITLE
Resolution of displayed values in bargraphs decreasing as ranges increase

### ISSUE THREAD

**Issue by dariosanfilippo** [grame-cncm/faust#322]
I use a signal inspector such as:

inspect(i, lower, upper) = _ <: attach(_ , _ : vbargraph("sig_%i [style:numerical]", lower, upper));.

The problem is that the resolution of the displayed value is inversely proportional to the size of the range.

For example, if the range is [-100;100], the decimal part is entirely gone.

Personally, I would love to have double precision displayed regardless of the range.

Thanks

**Comment by sletz** [comment]
Thanks. 
This the using QT Framework base generated applications right? (like `faust2caqt`, `faust2jaqt`....)

**Comment by dariosanfilippo** [comment]
Correct: I tested that with faust2caqt -double. Perhaps the precision option passed to the compiler could determine the precision of the displayed values.
